### PR TITLE
Fix not pruned bias in depthwise convolution

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build

File: tests/pruning/helpers.py [hunk 1]
```diff
@@ -39,16 +39,16 @@ class TestModelDiffConvs(nn.Module):
     def __init__(self):
         super().__init__()
         # Usual conv
-        self.conv1 = create_conv(1, 3, 2, 9, -2)
+        self.conv1 = create_conv(1, 32, 2, 9, -2)
```
Comment (mkaglins) [inline tests/pruning/helpers.py:42]: Why do you need to change shapes here?

Comment (AlexanderDokuchaev) [reply tests/pruning/helpers.py:42]: To check shapes in pruned module, number of remaining channels should be multiple of 8.

**Review by mkaglins — APPROVED** [review]

**Review by mkaglins — APPROVED** [review]

**Pull request merged**